### PR TITLE
LibWeb: Re-implement checkbox painting using the UA stylesheet

### PR DESCRIPTION
**Pull request by linusg**
Before and after:

image

image

### CODE REVIEW

File: Userland/Libraries/LibWeb/CSS/Default.css [hunk 1]
```diff
@@ -64,6 +64,22 @@ button:hover, input[type=submit]:hover, input[type=button]:hover, input[type=res
...
+    display: inline-block;
+    width: 12px;
+    height: 12px;
+    margin: 2px;
+    border: 1px solid -libweb-palette-threed-shadow1;
+}
+
+input[type=checkbox]:checked {
+    /* This roughly resembles ClassicStylePainter's paint_check_box() while uncoupling the styling from LibGfx,
+       similar to <button> above. This is a simple checkmark that still looks ok at 2x scale, eventually we
```
Comment (nico) [inline Userland/Libraries/LibWeb/CSS/Default.css:77]: nit: replace `,` with `.` as the part after it is unrelated to the part before it

**Review by nico — APPROVED** [review]
Nice!

**Comment by nico** [comment]
Also, might want to credit Xexxa for the bitmap in the commit message.

File: Userland/Libraries/LibWeb/CSS/Default.css (hunk 1 above)
Comment (linusg) [reply Userland/Libraries/LibWeb/CSS/Default.css]: Done, slightly expanded the comment, and credited Xexxa. Thanks!
